src/components/TagActions/Node/TagNode.tsx

```
@@ -0,0 +1,207 @@
+
+  return (
+    <div className="my-2">
+      <div>
+        {tag.childTags.totalCount ? (
+          <>
+            <span
+              onClick={handleTagClick}
+              className={`me-3 ${tagNodeStyles.cursorPointer}`}
+              data-testid={`expandSubTags${tag._id}`}
+              aria-label={expanded ? t('collapse') : t('expand')}
+            >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)